lib/axiom/fs/js/file_system.js

```diff
@@ -228,6 +228,16 @@ JsFileSystem.prototype.alias = function(pathFrom, pathTo) {
  * @param {Path} toPath
  * @return {!Promise<undefined>}
  */
+JsFileSystem.prototype.copy = function(fromPath, toPath) {
+  return this.alias(fromPath, toPath);
```

Comment:
Alias is a symbolic link semantics more than copy semantics, no? Why do we need to override the implementation?